packages/node/src/rpc/index.ts

```diff
@@ -70,7 +77,11 @@ export function init(node: DRPNode) {
 		callback(null, response);
 	}
 
+	const packageDefinition = protoLoader.loadSync(PROTO_PATH);
+	const reflectionService = new reflection.ReflectionService(packageDefinition);
+
 	const server = new grpc.Server();
+	reflectionService.addToServer(server);
```

Comment:
```suggestion
	server.addService(reflectionService);
```

does something like this work? to keep consistency

Reply:
let it be then